mollie.php

```diff
@@ -1316,7 +1316,7 @@ public function getTabs()
     public function hookActionAdminOrdersListingFieldsModifier($params)
     {
         if (isset($params['select'])) {
-            $params['select'] .= ' ,mol.`transaction_id`';
+            $params['select'] = rtrim($params['select'], ' ,') . ' ,mol.`transaction_id`';
```

Comment:
what if comma has no space before? It will not trim it.
However if there is a space and its not trimmed it shouldn't be a problem :thinking:
So it should be safer to simply trim `','` instead of `' ,'`

Reply:
I suppose you are not sure the same 'select' comes all the time and removing the comma from `' ,mol.`transaction_id`'` isn not an option? :smile:

Reply:
Tried out if it will actually not trim
`$testing4 = 'test,  ';
$testing5 = 'test    ,';
$testing6 = 'test,';
`
All these should be trimmed down to "test".

Yes, removing comma from mol.transaction_id is not an option

Reply:
> $testing4 = 'test, '; $testing5 = 'test ,'; $testing6 = 'test,';
All these should be trimmed down to "test"

So you just approved that your code will not work because you will only trim this pattern - `' ,'` (space and comma).
Try this:
rtrim($string, ", \t\n");